main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -1276,14 +1276,20 @@ private void updateWatchlistBox() {
             final Button buttonRemove = ButterKnife.findById(view, R.id.remove_from_watchlist);
             final TextView text = ButterKnife.findById(view, R.id.watchlist_text);
 
+            final int watchListCount = cache.getWatchlistCount();
+            String watchListCountString = "";
```

Comment:
Non-`final` variables are error-prone and should be avoided. Could you use something like this instead?

``` java
   final String watchListCountString = watchListCount != -1 ? '(' + … + ')' : "";
```